### PR TITLE
Set NetworkManager as a requirement

### PR DESCRIPTION
**Pull request by qinqon**
Now that kubernetes-nmstate is supported it needs NetworkManager to be
installed on the hosts, this patch document that and also add the
installation process at make cluster-up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,11 @@ The operator allows the administrator to deploy the [NmState State Controller](h
 This project manages host networking settings in a declarative manner. The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both conform to the schema.
 Nmstate is aimed to satisfy enterprise needs to manage host networking through a northbound declarative API and multi provider support on the southbound. NetworkManager acts as the main (and currently the only) provider supported.
 
+It communicate with a NetworkManager instance running on the node using D-Bus. Make sure that NetworkManager is installed and running on each node.
+
+yum install NetworkManager
```
Comment (SchSeba) [inline README.md]: Can you please add this as a code section?

File: cluster/up.sh [hunk 2]
```diff
@@ -68,3 +68,10 @@ echo 'Wait until all nodes are ready'
 until [[ $(./cluster/kubectl.sh get nodes --no-headers | wc -l) -eq $(./cluster/kubectl.sh get nodes --no-headers | grep ' Ready' | wc -l) ]]; do
     sleep 1
 done
+
+echo 'Install NetworkManager on nodes'
+for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
+    ./cluster/cli.sh ssh "node$(printf "%02d" ${i})" -- sudo yum install -y NetworkManager NetworkManager-ovs
```
Comment (phoracek) [inline cluster/up.sh]: We don't need NetworkManager-ovs now, please drop it.

File: README.md (hunk 1 above)
Comment (qinqon) [reply README.md]: Done thanks!

File: cluster/up.sh (hunk 2 above)
Comment (qinqon) [reply cluster/up.sh]: Bad copy paste :-/, done.

File: README.md [hunk 3]
```diff
@@ -143,6 +143,13 @@ The operator allows the administrator to deploy the [NmState State Controller](h
 This project manages host networking settings in a declarative manner. The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both conform to the schema.
 Nmstate is aimed to satisfy enterprise needs to manage host networking through a northbound declarative API and multi provider support on the southbound. NetworkManager acts as the main (and currently the only) provider supported.
 
+It communicate with a NetworkManager instance running on the node using D-Bus. Make sure that NetworkManager is installed and running on each node.
+
+~~~~
```
Comment (phoracek) [inline README.md]: We use a different syntax, please check the other examples.

File: cluster/up.sh [hunk 4]
```diff
@@ -68,3 +68,10 @@ echo 'Wait until all nodes are ready'
 until [[ $(./cluster/kubectl.sh get nodes --no-headers | wc -l) -eq $(./cluster/kubectl.sh get nodes --no-headers | grep ' Ready' | wc -l) ]]; do
     sleep 1
 done
+
+echo 'Install NetworkManager on nodes'
+for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
+    ./cluster/cli.sh ssh "node$(printf "%02d" ${i})" -- sudo yum install -y NetworkManage
```
Comment (phoracek) [inline cluster/up.sh]: NetworkManage**r**

File: README.md (hunk 3 above)
Comment (qinqon) [reply README.md]: ack, "```shell" it is

File: cluster/up.sh (hunk 4 above)
Comment (qinqon) [reply cluster/up.sh]: done

**Comment by phoracek** [comment]
LGTM, let's just wait for CI

**Pull request merged**